lib/remote-objects.js

```diff
@@ -93,7 +93,9 @@ RemoteObjects.extend = function(exports) {
 
 RemoteObjects.prototype.handler = function(name, options) {
   var Adapter = this.adapter(name);
-  var adapter = new Adapter(this, options);
+  // send the set options for this remote object and
+  // the dynamic options passed in to create the adapter
+  var adapter = new Adapter(this, Object.assign({}, this.options, options));
```

Comment:
This does not look right to me. Remoting options are different from transport/adapter-specific options. Why is this change needed? Can we revert it please?

Reply:
OK Bajtos. Thanks for the heads up. I got a neater solution using the actual Rest properties that doesn't need config code changes. the 'options.rest' config goes directly to the rest adapter already.